samples/notebooks/quickstart.ipynb

```diff
@@ -0,0 +1,572 @@
+    "args = parser.parse_args()\n",
+    "# Create a client.\n",
+    "storage_client = storage.Client()\n",
+    "# List blobs.\n",
+    "bucket = storage_client.get_bucket(args.bucket)\n",
+    "list_blobs_response = bucket.list_blobs()\n",
+    "blobs = ','.join([blob.name for blob in list_blobs_response])\n",
+    "print(blobs)\n",
+    "with open('/blobs.txt', 'w') as f:\n",
+    "  f.write(blobs)\n",
+    "HERE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now create a container that runs the script. Start by creating a `Dockerfile`. A `Dockerfile` contains the instructions to assemble a Docker image. The `FROM` statement specifies the Base Image from which you are building. `WORKDIR` sets the working directory. When you assemble the Docker image, `COPY` will copy the required files and directories (for example, `app.py`) to the filesystem of the container. `RUN` will execute a command (for example, install the dependencies) and commits the results. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "# Create Dockerfile.\n",
+    "cat > ./tmp/components/list-gcs-blobs/Dockerfile <<HERE\n",
+    "FROM python:3.6-slim\n",
+    "WORKDIR /app\n",
+    "COPY . /app\n",
+    "RUN pip install --upgrade google-cloud-storage\n",
+    "HERE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now create a Shell script that builds a container image and stores it in the Google Container Registry (GCR)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash -s \"{PROJECT_ID}\"\n",
+    "\n",
+    "IMAGE_NAME=\"listgcsblobs\"\n",
+    "TAG=\"latest\" # \"v_$(date +%Y%m%d_%H%M%S)\"\n",
+    "\n",
+    "# Create script to build docker image and push it.\n",
+    "cat > ./tmp/components/list-gcs-blobs/build_image.sh <<HERE\n",
+    "PROJECT_ID=\"${1}\"\n",
+    "IMAGE_NAME=\"${IMAGE_NAME}\"\n",
+    "TAG=\"${TAG}\"\n",
+    "GCR_IMAGE=\"gcr.io/\\${PROJECT_ID}/\\${IMAGE_NAME}:\\${TAG}\"\n",
+    "docker build -t \\${IMAGE_NAME} .\n",
+    "docker tag \\${IMAGE_NAME} \\${GCR_IMAGE}\n",
+    "docker push \\${GCR_IMAGE}\n",
+    "docker image rm \\${IMAGE_NAME}\n",
+    "docker image rm \\${GCR_IMAGE}\n",
+    "HERE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the script."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "# Build and push the image.\n",
+    "cd tmp/components/list-gcs-blobs\n",
+    "bash build_image.sh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.2 Create a Python class for each component\n",
```

Comment:
This doesn't have to be a Python class. It can also be a function. I'd rather not promote this way of making container ops, especially as we're moving towards the direction where python functions define the interface to a component. So, for example:

def list_gcs_blobs(name, bucket):
  return dsl.ContainerOp(....)

Reply:
Done!